.drone.yml

```diff
@@ -0,0 +1,31 @@
+kind: pipeline
+type: docker
+name: default
+
+steps:
+  - name: fetch
+    image: alpine/git
+    commands:
+      - git fetch --tags
+    when:
+      event:
+        - tag
+  - name: build
+    image: larcher/manylinux-golang
```

Comment:
@lbn leftover: please re-push image to `scalrlabs` account (creds are in confluence), and include it's Dockerfile into the repo.